src/app/loadout-drawer/LoadoutDrawer2.tsx

```diff
@@ -294,17 +302,24 @@ function filterLoadoutToAllowedItems(
   });
 }
 
-async function pickLoadoutItem(
-  defs: D1ManifestDefinitions | D2ManifestDefinitions,
-  loadout: Loadout,
-  bucket: InventoryBucket,
-  add: (item: DimItem) => void,
-  onShowItemPicker: (shown: boolean) => void
-) {
+async function pickLoadoutItem({
```

Comment:
Why change this to be called with an object?

Reply:
It was just a preference for a named parameters style destructing - happy to change it back though!

Reply:
Yeah probably best not to change stuff unrelated to the direct fix.